src/runtime/storageNG/drivers/tests/volatile-test.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * @license
+ * Copyright (c) 2019 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {assert} from '../../../../platform/chai-web.js';
+import {VolatileDriver, VolatileStorageKey} from '../volatile.js';
+import {Exists} from '../driver-factory.js';
+import {Runtime} from '../../../runtime.js';
+
+describe('Volatile Driver', async () => {
+
+  afterEach(() => {
+    Runtime.clearRuntimeForTesting();
+  });
+
+  it('can be multiply instantiated against the same storage location', () => {
+    const volatileKey = new VolatileStorageKey('unique');
+    const volatile1 = new VolatileDriver(volatileKey, Exists.ShouldCreate);
+    const volatile2 = new VolatileDriver(volatileKey, Exists.ShouldExist);
```

Comment:
Assert that `volatile1.data === volatile2.data` ?  (would need an unsafe() wrapper)

Reply:
I'm torn on this one; I can see why you want it I think (there's no explicit assert here, we're just checking for no exceptions), but it seems like a violation of black-box testing and one that isn't necessary given the other tests cover the functional intent of that check (i.e. that multiple drivers with the same key modify the same value).

Reply:
Sounds fair.